docs/src/release_notes/v1.21.md

```diff
@@ -6,16 +6,79 @@ For a complete list of changes, please refer to the
 [commits](https://github.com/cloudnative-pg/cloudnative-pg/commits/release-1.21)
 on the release branch in GitHub.
 
-## Version 1.21.5
+## Version 1.21.6
 
-**Release date:** Apr 24, 2024
+**Release date:** Jun 12, 2024
 
 !!! Warning
-    Version 1.21 is approaching its End-of-Life (EOL) on May 24, 2024.
-    If you haven't already, please begin planning for an upgrade promptly to
-    ensure continued support and security.
+    This is expected to be the last release in the 1.21.X series.
+    Users are encouraged to update to a newer minor version soon.
+
+### Enhancements:
+
+- Enabled configuration of standby-sensitive parameters during recovery from a
+  physical backup (#4564)
+
+- Enabled the configuration of the liveness probe timeout via the
+  `.spec.livenessProbeTimeout` option (#4719)
+
+- `cnpg` plugin for `kubectl`:
+
+    - Introduced support for ANSI colors in the plugin through the `--color`
+      option, which accepts `always`, `never`, and `auto` (default) as values
+      (#4775)
+    - The plugin is now available on Homebrew for macOS users (#4602)
+
+### Fixes:
+
+- Prevented fenced instances from entering an unnecessary loop and consuming
+  all available CPU (#4625)
+
+- Resolved an issue where the instance manager on the primary would
+  indefinitely wait for the instance to start after encountering a failure
+  following a stop operation (#4434)
+
+- Fixed an issue where the interaction between `hotstandby_feedback` and
+  managed cluster-level replication slots was preventing the removal of
+  obsolete replication slots; this issue was causing disk space to remain
+  occupied by dead tuples (#4811)
```

Comment:
This is not my understanding of the issue.
```suggestion
  managed cluster-level replication slots was wrongly setting xmin on
  standby slots, which was then set on the primary and prevented VACUUM
  from removing dead tuples and freeing disk space (#4811)
```

Reply:
I think both versions are wrong. It's not true that `hot_standby_feedbak` was preventing the removal of obsolete replication slot, not it sets the `xmin` on standby. It only propagates the `xmin` from the standby (where it was harmless) to the primary. I would not go into details and rater say:

```suggestion
  managed cluster-level replication slots was preventing the autovacuum from operate correctly; this issue was causing disk space to remain
  occupied by dead tuples (#4811)
```

Reply:
s/from operate/from operating/
and, line seems very long, otherwise 👍 on that, Marco.

Reply:
updated